rh/management/commands/load_fictive_data.py

```diff
@@ -0,0 +1,550 @@
+import random
+import ast
+import datetime
+
+import django
+django.setup()
+
+from django.contrib.auth.models import User
+
+from django.contrib.auth import get_user_model
+
+import faker.providers
+from django.core.management.base import BaseCommand
+
+from faker import Faker
+from ...models import *
+
+COUNTRIES = [ 
```

Comment:
This is an overkill. what we need is no more than 2 countries (Afghanistan and Somalia). I see further below that you are inserting randomised countries but it is not necessary. This is only a test script which is going to be removed eventually. we should keep it as simple as possible.

Reply:
Done